CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Unreleased
 
+- BREAKING: Remove graphql App, Shop, AuthToken server.
+  - If you want this/use this, grab it out of the git history and import in to your project.
```

Comment:
Is this heads-up needed for anyone in particular?

Reply:
Just anyone using it, also as a not for me to remember to go through our apps when we upgrade to this ver.